### PR TITLE
CLN:Fix flake8 errors on test/test_compare.py file 

### LINKED ISSUE #1042: Fix Flake8 violations for all files in the test/ directory
We are trying to clean out all the files to follow Flake8 standards

Use this guide below to install flake8 and how to use it.
[ Flake8 guide](https://siderlabs.com/blog/automatically-review-code-for-python-projects-using-flake8-6fcb056a001a/)

Here are a few steps to follow:
- You will be required to remove the files you would like to work on from the `setup. cfg` file under the `extend` attribute .
- Create a virtual environment `virtualenv venv` venv will be the name for the environment and activate it by running `source venv/bin/activate` these commands are dependent on the operating system. (These are Linux commands, you can find for windows and macOS online)
- Run `pip install flake8` to install flake8
- cd test
- Run `flake8 --exclude="" filename`  for each file in the test directory
- Work on all the highlighted errors

### LINKED ISSUE #1165: [Sprint] Fix flake8 violations for files on test/test_compare.py
We are trying to clean out all the files to follow Flake8 standards.

Here are a few steps to follow:
- Create a virtual environment virtualenv venv venv will be the name for the environment and activate it by running source venv/bin/activate these commands are dependent on the operating system. (These are Linux commands, you can find for windows and macOS online).
- Run `pip install flake8` to install flake8.
- Run `flake8 --exclude="" test/test_compare.py` .
- Work on all the highlighted errors.
- At the end, you will be required to remove the files you would like to work on from the setup. cfg file under the exclude attribute.

For this file you will find the following issues to solve
```
test/test_compare.py:101:30: W291 trailing whitespace
test/test_compare.py:119:100: E501 line too long (108 > 99 characters)
test/test_compare.py:120:100: E501 line too long (100 > 99 characters)
test/test_compare.py:121:100: E501 line too long (102 > 99 characters)
test/test_compare.py:122:100: E501 line too long (108 > 99 characters)
test/test_compare.py:123:100: E501 line too long (114 > 99 characters)
test/test_compare.py:125:100: E501 line too long (103 > 99 characters)
test/test_compare.py:126:100: E501 line too long (107 > 99 characters)
test/test_compare.py:129:100: E501 line too long (102 > 99 characters)
```

### PR DESCRIPTION
**Pull request by dorothykiz1**
close #1042
close #1165

### CODE REVIEW

**Comment by dorothykiz1** [comment]
@datapythonista this PR is ready for your review

**Comment by datapythonista** [comment]
Looks good, there is a conflict.

**Comment by dorothykiz1** [comment]
@datapythonista this PR is ready for your review

File: setup.cfg [hunk 1]
```diff
@@ -64,10 +64,7 @@ max-line-length = 99
 ignore =
     W504,  # W504: Line break occurred after a binary operator
     E741  # E741: Do not use variables named 'I', 'O', or 'l'
-exclude =
-        asv/extern,
-        test/test_compare.py
-
+exclude = asv/extern
```
Comment (datapythonista) [inline setup.cfg:67]: Can you leave a blank line before the next section please.

File: test/test_compare.py [hunk 2]
```diff
@@ -129,7 +129,7 @@
 +           1.00s            3.00s     3.00  time_with_version_match [cheetah/py2.7-numpy1.8]
 +           1.00s            3.00s     3.00  time_with_version_mismatch_bench [cheetah/py2.7-numpy1.8]
 -          69.1μs           18.3μs     0.27  time_units.time_unit_to [cheetah/py2.7-numpy1.8]
-"""
+""" # noqa :E501 line too long
```
Comment (datapythonista) [inline test/test_compare.py:132]:
```suggestion
""" # noqa: E501 line too long
```

**Comment by datapythonista** [comment]
Thanks @dorothykiz1, really nice, no more pep8 errors :)

**Pull request merged**